WalletWasabi.Fluent/WalletWasabi.Fluent.csproj

```diff
@@ -24,7 +23,7 @@
 		<PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
 		<PackageReference Include="Avalonia.Controls.TreeDataGrid" Version="0.10.10" />
 		<PackageReference Include="Avalonia.ReactiveUI" Version="$(AvaloniaVersion)" />
-		<PackageReference Include="Avalonia.Diagnostics" Version="$(AvaloniaVersion)" />
+		<PackageReference Include="Avalonia.Diagnostics" Version="$(AvaloniaVersion)" Condition="'$(Configuration)' == 'Debug'" />
```

Comment:
Breaking the packager? https://github.com/zkSNACKs/WalletWasabi/issues/6679

Reply:
we should not have dependency on this in the release build.

It adds huge deps: MSBuild, Roslyn compiler, Dotnet refactoring and code analysis tools... this will add huge amounts of code to the final package..

Reply:
This can come back but only with this https://github.com/zkSNACKs/WalletWasabi/pull/6881

Reply:
This is breaking the current code. Remove this change and elaborate on it with @kiminuo 's help.

Reply:
ok ill revert that for now.. but really need to remove that at some point.

Reply:
removed.